vendor/github.com/openshift-kni/performance-addon-operators/functests/4_latency/latency.go

```diff
@@ -0,0 +1,257 @@
+package __latency
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	v1 "github.com/openshift-kni/performance-addon-operators/api/v1"
+	testutils "github.com/openshift-kni/performance-addon-operators/functests/utils"
+	testclient "github.com/openshift-kni/performance-addon-operators/functests/utils/client"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/discovery"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/nodes"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/pods"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/profiles"
+	"github.com/openshift-kni/performance-addon-operators/pkg/controller/performanceprofile/components"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+	"k8s.io/kubernetes/pkg/kubelet/cm/cpuset"
+	"k8s.io/utils/pointer"
+)
+
+var (
+	latencyTestDelay   = "0"
+	latencyTestRun     = false
+	latencyTestRuntime = "300"
+	latencyTestImage   = "quay.io/openshift-kni/oslat:latest"
+	maximumLatency     = -1
+)
+
+// LATENCY_TEST_RUN: indicates if the latency test should run
+// LATENCY_TEST_RUNTIME: the amount of time in seconds that the latency test should run
+// LATENCY_TEST_IMAGE: the image use under the latency test
```

Comment:
I know this is vendored but I'll comment it here:
- can we split it between registry and image name? It will help us for the mirroring/disconnected part.
- also, can we point to the cnf-tests image?

Reply:
Once we will merge this one, I will change the latency test to use only the cnf-tests.